test_pkm.py

```
@@ -0,0 +1,168 @@
+    # Test job execution
+    success = scheduler.run_job("test_job")
+    console.print(f"✅ Test job executed: {success}")
+    
+    # Remove test job
+    removed = scheduler.remove_job("test_job")
+    console.print(f"✅ Test job removed: {removed}")
+
+
+async def main():
+    """Main test function."""
+    console.print(Panel.fit(
```

Ruff (0.13.2)

Undefined name `Panel`

(F821)